react/ui-components/src/hoc/app-config-screen-builder/AppConfigurationWrapper.js

```
@@ -0,0 +1,863 @@
+            <DndProvider backend={HTML5Backend}>
+              <AppFieldScreenWrapper />
+            </DndProvider>
+          )}
+        </SidePanel>
+      )}
+      {showPreview && (
+        <PopUp
+          className={"custom-pop-up"}
+          type={"default"}
+          heading={t("CHECKLIST_PREVIEW")}
+          children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)